package.json

```diff
@@ -33,13 +33,13 @@
     "react-static-container": "^1.0.0-alpha.1"
   },
   "peerDependencies": {
+    "babel-relay-plugin": "^0.3.0",
     "react": "^0.14.0",
     "react-dom": "^0.14.0"
   },
   "devDependencies": {
     "babel-core": "^5.8.25",
     "babel-eslint": "^4.1.1",
-    "babel-relay-plugin": "^0.3.0",
```

Comment:
Both the tests and the examples use the local plugin from `/scripts/babel-relay-plugin` - nothing in dev mode requires the published version of the plugin anymore.